common/OpTestSystem.py

```diff
@@ -1008,7 +1008,7 @@ def sys_ipmi_boot_system_to_petitboot(self, i_console):
 
         # Exiting to petitboot shell
         self.console.expect('Petitboot', timeout=BMC_CONST.PETITBOOT_TIMEOUT)
-        time.sleep(20)
+        self.console.expect('x=exit', timeout=10)
```

Comment:
Here we may face one more problem, we are reaching to petitboot UI early than actual running of petitboot boot discovery server. So if we immediately execute any command it hangs for some commands which are require any disk/network/dvd/usb info as the pb-discover server not at discovered all the devices, as we just reached to UI .  But as of now it is okay for setup to petitboot. In the test case, based on the command we need to wait for some time or is any debug hook available such that user is notified about pb-discovery server running completion. So we can wait until then.